bin/delete_user_data.py

```diff
@@ -0,0 +1,91 @@
+#
+# A helper script to delete user data from a Sync storage server.
+#
+# You can use this script to explicitly delete stored sync data
+# for a user, without having to connect a Firefox profile and
+# without having to reset their password.  It may be useful if
+# you've started running a self-hosted storage server and want
+# to delete data that was previously stored on the Mozilla-hosted
+# servers.
+#
+# Use it like so:
+#
+#    $> pip install PyFxA
+#    $> python delete_user_data.py user@example.com
+#
+# The script makes a best-effort attempt to sign in to the user's
+# account, authenticate to the Firefox Sync Tokenserver, and delete
+# the user's stored sync data.  The login process might fail due to
+# things like rate-limiting, server-side security measures, or API
+# changes in the login process.
+#
+
+import sys
+import getpass
+import hashlib
+import argparse
+import urlparse
+
+import requests
+import hawkauthlib
+import fxa.core
+
+DEFAULT_FXA_URI = "https://api.accounts.firefox.com"
+DEFAULT_TOKENSERVER_URI = "https://token.services.mozilla.com"
+
+def main(argv):
+    parser = argparse.ArgumentParser(description="Delete Firefox Sync data")
+    parser.add_argument("email",
+                        help="account for which to delete data")
```

Comment:
nit: Email of the account for which to delete data?

Reply:
lol, I had actually written that originally, but it went over 80 columns wide so I edited it down.  That's not a good reason to make the help text less clear, so I'll add it back.  Thanks!